internal/cmd/grpc.go

```diff
@@ -187,6 +194,7 @@ func NewGRPCServer(
 		ctx,
 		logger,
 		cfg.Authentication,
+		auditLoggingEnabled,
```

Comment:
@GeorgeMac Yeah right now it will work. In the future it would be if any sink is enabled then audit logging is enabled in general. That is why I abstracted that bit into a variable if that makes sense?